src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -309,8 +309,7 @@ $(BashLinkerTestCleanupCmds)
       ]]></BashCLRTestLaunchCmds>
       <BashCLRTestLaunchCmds Condition="'$(CLRTestKind)' == 'BuildAndRun' And $(TargetOS) == 'Browser' ">
       <![CDATA[
-cd WasmApp
-./run-v8.sh
+cd WasmApp && ./run-v8.sh
 CLRTestExitCode=$?
```

Comment:
Is there any chance this will look like a pass if the cd fails with the same error code the test was expected to fail with?

Reply:
I'm not sure about that. Though there is a difference between the exit codes in the two cases:

1.
```sh
$ (cd bad; ./qwe); echo $?
bash: cd: bad: No such file or directory
bash: ./qwe: No such file or directory
127
```

2.
```sh
$ (cd bad && ./qwe); echo $?
bash: cd: bad: No such file or directory
1
```

I'm not sure where the expected codes are being set by tests.
I can remove this change, so we don't break anything.

Reply:
The tests, and thus the run-v8.sh command, all return 100 on success. So, its possible that if cd returns 100 as an error code on some condition, it could appear to pass. I don't know if there is any case where cd does that.

Reply:
This specific change is just a "nicety", so I can remove it.

Reply:
I think it is pretty unlikely to happen, but it would be really hard to notice if it ever did happen. So we probably should not have this change.